### PR TITLE
Fix for boot error for CardKB on ESP32 Heltec V3

### PR DESCRIPTION
**Pull request by HarukiToreda**
Fix for "[Bug]: CardKB not starting on ESP32 when first powered #3749"

I introduced a slight delay to allow the Atmega chip on the CardKB to boot properly and be ready for when the I2C scan happens on the ESP32. With this, the error of "Canned Message disabled" is fixed.

### CODE REVIEW

**Comment by HarukiToreda** [comment]
If anyone could please help with the trunk error, this has been tested on the Heltec V3, WSL and the tracker, I just have no not been able to fix this Trunk error

File: .trunk/configs/.clang-format [hunk 1]
```diff
@@ -3,4 +3,4 @@ IndentWidth: 4
 ColumnLimit: 130
 PointerAlignment: Right
 BreakBeforeBraces: Linux
-AllowShortFunctionsOnASingleLine: Inline
+AllowShortFunctionsOnASingleLine: Inline
```
Comment (caveman99) [inline .trunk/configs/.clang-format:6]: Please revert the changes to the .trunk and .vscode files - these changes are unrelated to this PR

File: src/main.cpp [hunk 2]
```diff
@@ -391,6 +391,7 @@ void setup()
 
     // We need to scan here to decide if we have a screen for nodeDB.init() and because power has been applied to
     // accessories
+    delay(500);  // Introduce a delay before scanning for I2C devices
```
Comment (caveman99) [inline src/main.cpp:394]: Alternate suggestion: repeat the bus scan/test for keyboards in the cannedMessages init. This 500msec will go on top the 5s delay we have waiting for the serial port.

**Review by caveman99 — CHANGES_REQUESTED** [review]

File: src/main.cpp (hunk 2 above)
Comment (HarukiToreda) [reply src/main.cpp:394]: I added a way to rescan right after and it works. Removed the delay

Comment (HarukiToreda) [reply src/main.cpp:394]: nvm, it gave me lots of errors. reverting to delay

**Comment by caveman99** [comment]
My i add a suggestion to your repo? so you can compile&test. That will rescan only the known keyboard addresses and not all of the bus.

**Comment by HarukiToreda** [comment]
I have not been able to figure out a different way, not sure if you'd prefer this closed for someone else to try someday.
